.buildkite/pipeline.yml

```diff
@@ -9,39 +9,78 @@ plugins: &plugins
     mode: checkout
 
 steps:
+  - label: ":docker::python: build 3.6 runner"
+    key: "runner-3_6"
+    plugins:
+      <<: *plugins
+      docker-compose#v3.1.0:
+        <<: *compose-config
+        build: runner-3_6
+        image-repository: index.docker.io/stellargraph/stellargraph-ci-runner
+        no-cache: ${REFRESH_BUILD-false}
+        cache-from:
+          - runner-3_6:index.docker.io/stellargraph/stellargraph-ci-runner:3_6-${IMAGE_TAG}
+          - runner-3_6:index.docker.io/stellargraph/stellargraph-ci-runner:3_6-latest
+
```

Comment:
I assume this whitespace is intentional for readability?

Reply:
Yeah, it's separating the two separate sub-steps; above is `build`, and below is `push`.